packages/next/build/babel/preset.ts

```diff
@@ -157,6 +157,8 @@ module.exports = (
           helpers: true,
           regenerator: true,
           useESModules: supportsESM && presetEnvConfig.modules !== 'commonjs',
+          // the lower of the entries in package.json for `@babel/rutime` and `@babel/runtime-corejs2`
+          version: '7.7.2',
```

Comment:
Reading this from the current package.json is probably smarter. I don't know how your deployed file layout looks like or if you already have functionality for this. If you can give directions I'll happily improve this.

Reply:
`require('@babel/runtime/package.json').version`

Reply:
That's perfect. It even works with overriding the `@babel/runtime` version via `resolutions`.